src/conf_mode/container.py

```diff
@@ -40,6 +41,7 @@
 
 config_containers_registry = '/etc/containers/registries.conf'
 config_containers_storage = '/etc/containers/storage.conf'
+config_containers_auth = '/etc/containers/auth.json'
```

Comment:
Please store the authfile in a volatile location like /run/podman so it's cleaned on reboot. Both other files are cleaned in a hook: https://github.com/vyos/vyatta-cfg/blob/current/scripts/init/vyos-router#L233 or add that file to the hook.